app/[locale]/ledger/income/IncomeFormPayment.tsx

```diff
@@ -0,0 +1,84 @@
+'use client'
+
+import ErrorSummary from "@/app/components/ErrorSummary"
+import RequiredFieldDescription from "@/app/components/RequiredFieldDescription"
+import TextFieldWithValidation from "@/app/components/TextFieldWithValidation"
+import { PaymentItem } from "@/lib/features/ledger/income/incomeSlice"
+import { Button, DatePicker, Form, FormGroup, Label, RequiredMarker } from "@trussworks/react-uswds"
+import { Controller, SubmitHandler, useForm } from "react-hook-form"
+import { useTranslation } from "react-i18next"
+
+export interface IncomeFormPaymentProps {
+    onSubmit: SubmitHandler<IncomeFormPaymentData>
+    item?: PaymentItem
+}
+
+export type IncomeFormPaymentData = {
+    amount: number
+    date: string
+    payer: string
+}
+
+export default function IncomeFormPayment(params: IncomeFormPaymentProps) {
+    const { t } = useTranslation()
+
+    const {
+        register,
+        control,
+        formState: { errors },
+        handleSubmit
+    } = useForm<IncomeFormPaymentData>()
+
+    return (
+        <Form onSubmit={handleSubmit(params.onSubmit)}>
+            <RequiredFieldDescription />
+            <ErrorSummary errors={errors} headingText={t('add_income_error_header')} />
+
+            <FormGroup>
+                <TextFieldWithValidation
+                    id="amount"
+                    {...register("amount", { valueAsNumber:true, validate: (value) => value > 0, required: { value: true, message: t('add_income_payment_amount_field_required')}})}
+                    label={t('add_income_payment_amount')}
+                    error={errors.amount?.message}
+                    requiredMarker={true}
+                    data-testid="amount"
+                    value={params.item?.amount?.toString()}
+                />
+            </FormGroup>
+
+            <FormGroup>
+                <Controller
+                    name="date"
+                    control={control}
+                    rules={{ required: {value:true, message: t('add_income_required_field')} }}
+                    render={({ field }) => (
+                        <>
+                            <Label htmlFor="date">{t('add_income_payment_date')}<RequiredMarker /></Label>
```

Comment:
The other labels have the `text-bold` css class.
<img width="396" alt="Screenshot 2024-09-20 at 1 17 27 PM" src="https://github.com/user-attachments/assets/2b6e5736-1608-46c0-9402-a857986a4313">

Reply:
ooh good catch